builtin/settingtypes.txt

```diff
@@ -1301,9 +1301,7 @@ mapgen_limit (Map generation limit) int 31000 0 31000
 #    Global map generation attributes.
 #    In Mapgen v6 the 'decorations' flag controls all decorations except trees
 #    and junglegrass, in all other mapgens this flag controls all decorations.
-#    Flags that are not enabled are not modified from the default.
```

Comment:
It's ok to remove this line (and identical lines below), but no other changes should be made to settingtypes.txt.

Reply:
Okay. I deleted those because I only want to show the flag without "no" prefix. But, it seems like those are still needed and there is another way to do it.